### ISSUE TITLE
Code check should not warn "Unresolved reference uuid" for data onepassword_vault

### ISSUE THREAD

**Issue by mrclrchtr** [1Password/terraform-provider-onepassword#159]
### Summary
It's actually possible to use the `onepassword_vault` data source to read the `uuid` of a vault per `name`.

But the code check (IntelliJ in my case) shows the following error:

![image](https://github.com/1Password/terraform-provider-onepassword/assets/16151098/fcb533bc-12ac-4406-bbcb-6c24c5c21946)

"Unresolved reference uuid"

**Comment by SimonBarendse** [comment]
Hi @mrclrchtr thanks for reaching out and opening this issue!

I'm wondering whether this is a problem with the provider or with the code check / IntelliJ extension.

As you've mentioned :
> It's actually possible to use the `onepassword_vault` data source to read the `uuid` of a vault per `name`.

I believe this means the provider works as expected and [documented](https://registry.terraform.io/providers/1Password/onepassword/latest/docs/data-sources/item#uuid). Does this code indeed work when you `terraform plan`? If so, I think we need to move this issue to the extension you're using as this may be an issue in that extension rather than in this provider.

**Comment by mrclrchtr** [comment]
Yes, I think you are right. I'm using the provided extension from JetBrains.
